aten/src/ATen/native/native_functions.yaml

```diff
@@ -6217,6 +6217,18 @@
     CPU: trace_cpu
     CUDA: trace_cuda
 
+- func: linalg_trace(Tensor self, int offset=0) -> Tensor
+  python_module: linalg
+  variants: method, function
+  dispatch:
+    CPU, CUDA: linalg_trace
+    CompositeExplicitAutograd: linalg_trace
+
+- func: linalg_trace_backward(Tensor grad, int[] sizes, int offset) -> Tensor
```

Comment:
Could you please remove this entry from native_functions.yaml?
Most of the backward functions in PyTorch are placed in `torch\csrc\autograd\FunctionsManual.cpp` and `torch\csrc\autograd\FunctionsManual.h`, so let's move `linalg_trace_backward` from `ReduceOps.cpp`.